### PR TITLE
Updates Fireaxe.dm to include multitool method & map changes

### PR DESCRIPTION
**Pull request by OopsAllBrainWorms**
i ran mapmerger

[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/brushtool/fpstation/blob/master/fpstation/README.md )

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

## Why It's Good For The Game

thank you fuckynuggets for being passive aggressive :^)
and if i fucked this up yell at me some more cause that'll totally help it

### CODE REVIEW

**Comment by funkynuggets** [comment]
no, god no. People use the fire axe because it's easy, if you do this the next easy step is either using the SM as a handy instant open tool, or using water potassium grenades. The fire axe isn't even that good, the chef can make a knife that does the same damage. all you're doing is pushing people to use more extreme methods, such as emitters/handmade pulse guns/grenades/etc etc

**Comment by OopsAllBrainWorms** [comment]
@funkynuggets alright 300 intergrity on the fireaxe cabinet and 12 seconds to unlock?

**Comment by funkynuggets** [comment]
> 
> 
> @funkynuggets alright 300 intergrity on the fireaxe cabinet and 12 seconds to unlock?

I don't think you realize it's not the fire axe is the problem. It's only used because it's convienent, people can use all other methods, such as CLF3 fire, wat pot explosion, locker mech, ripley. People don't steal it for no reason, it's literally the keys for the city and is required when there's no cap/hop. Weisyl said it best, moving the caps office would work better.

File: _maps/map_files/FacepunchStation/facepunchstation.dmm [hunk 1]
```diff
@@ -21712,13 +21712,14 @@
...
 "bcd" = (
-/obj/structure/fireaxecabinet{
-	pixel_y = -32
-	},
 /obj/effect/turf_decal/tile/blue,
 /obj/effect/turf_decal/tile/blue{
 	dir = 8
 	},
+/obj/structure/fireaxecabinet/face{
+	step_x = 1;
```
Comment (ghost) [inline _maps/map_files/FacepunchStation/facepunchstation.dmm:21720]: pixel_x

File: _maps/map_files/FacepunchStation/facepunchstation.dmm [hunk 2]
```diff
@@ -21712,13 +21712,14 @@
...
+	step_y = -32
```
Comment (ghost) [inline _maps/map_files/FacepunchStation/facepunchstation.dmm:21721]: pixel_y

File: _maps/map_files/FacepunchStation/facepunchstation.dmm [hunk 3]
```diff
@@ -43586,13 +43587,13 @@
...
 "cgi" = (
-/obj/structure/fireaxecabinet{
-	pixel_x = -32
-	},
 /obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer1{
 	icon_state = "manifold-1";
 	dir = 8
 	},
+/obj/structure/fireaxecabinet/face{
+	step_x = -31
```
Comment (ghost) [inline _maps/map_files/FacepunchStation/facepunchstation.dmm:43595]: pixel_x

**Review by ghost — CHANGES_REQUESTED** [review]
step_x and step_y break the game
use pixel_x and pixel_y

**Pull request closed without merging**